syntax = "proto2";

package com.trivago.streaming.toolkit.proto.mysql.accommodation;

option java_outer_classname = "concept_scoreOuter";
option java_package = "com.trivago.streaming.toolkit.proto.mysql.accommodation";
option go_package = "com.trivago.streaming.toolkit.proto.mysql.accommodation";

message concept_score {
    message PK {
        optional uint32 concept_id = 1;
        optional uint32 concept_ns = 2;
        optional uint32 accommodation_id = 3;
        optional uint32 accommodation_ns = 4;
    }
    optional PK key = 2;

    optional uint32 score = 3;
}